dcpy/models/product/dataset/metadata_v2.py

```diff
@@ -48,50 +48,26 @@ class CustomizableBase(SortedSerializedBase, extra="forbid"):
 
 
 # COLUMNS
-# TODO: move to share with ingest.validate
-class Checks(CustomizableBase):
-    is_primary_key: bool | None = None
-    non_nullable: bool | None = None
-
-
-# TODO: move to share with ingest.validate
-COLUMN_TYPES = Literal[
-    "text", "number", "integer", "decimal", "geometry", "bool", "bbl", "datetime"
-]
-
-
 class ColumnValue(CustomizableBase):
     _head_sort_order = ["value", "description"]
 
     value: str
     description: str | None = None
 
 
-class DatasetColumn(CustomizableBase):
+class DatasetColumn(Column):
     _head_sort_order = ["id", "name", "data_type", "description"]
     _tail_sort_order = ["example", "values", "custom"]
-    _validate_data_type = (
-        True  # override, to generate md where we don't know the data_type
-    )
 
     # Note: id isn't intended to be overrideable, but is always required as a
-    # pointer back to the original column, so it is required here.
-    id: str
+    # pointer back to the original column.
     name: str | None = None
-    data_type: str | None = None
     data_source: str | None = None
-    description: str | None = None
     notes: str | None = None
     example: str | None = None
-    checks: Checks | None = None
     deprecated: bool | None = None
     values: list[ColumnValue] | None = None
-
-    @field_validator("data_type")
-    def _validate_colum_types(cls, v):
-        if cls._validate_data_type:
```

Comment:
Ohh this was not a pydantic arg, this was a custom one defined by @alexrichey. Sorry @sf-dcp I thought that was some sort of built-in pydantic behavior